### PR TITLE
keyboard shortcut to clear terminal logs

### PR DESCRIPTION
**Pull request by julien-c**
see #412

### CODE REVIEW

**Comment by dpgraham** [comment]
Looks good, just one comment.

**Comment by dpgraham** [comment]
@julien-c Do you have time to fix the one thing I commented on?

File: app/renderer/components/ServerMonitor/ServerMonitor.js [hunk 1]
```diff
@@ -94,6 +94,20 @@ export default class ServerMonitor extends Component {
...
+      this.props.clearLogs();
+    }
+  }
+
+  componentDidMount() {
+    document.addEventListener('keydown', this.keydownListener.bind(this));
+  }
+
+  componentWillUnmount() {
+    document.removeEventListener('keydown', this.keydownListener.bind(this));
```
[the remark this answers is not in the document]
Comment (jlipps) [reply app/renderer/components/ServerMonitor/ServerMonitor.js]: @dpgraham you still have `bind` here though?

[the remark this answers is not in the document]
Comment (dpgraham) [reply app/renderer/components/ServerMonitor/ServerMonitor.js]: Yeah that's a typo.

[the remark this answers is not in the document]
Comment (dpgraham) [reply app/renderer/components/ServerMonitor/ServerMonitor.js]: Fixed

**Comment by dpgraham** [comment]
I'll just merge this and then post a patch afterwards

File: app/renderer/components/ServerMonitor/ServerMonitor.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (julien-c) [reply app/renderer/components/ServerMonitor/ServerMonitor.js]: Good catch @dpgraham

**Comment by julien-c** [comment]
(Sorry I was out of town) Thanks for merging!

**Comment by dpgraham** [comment]
No problem. Thanks for the contribution!

**Comment by julien-c** [comment]
@dpgraham I'm looking at contributing into some lower-level areas of `appium` – any way to invite me to appium's slack? Thanks!

**Comment by dpgraham** [comment]
@julien-c Yeah absolutely! Our Slack channel is appium.slack.com... it's open for anyone to join.

**Comment by julien-c** [comment]
@dpgraham Are you sure? I see this when trying to create an account:
capture d ecran 2018-03-05 a 20 29 17

**Comment by saikrishna321** [comment]
can u please share ur email Id will send you invite

**Comment by julien-c** [comment]
No worries!

**Pull request merged**